# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/layers/apz/src/APZCTreeManager.h
# Commit: fe20f47f85d9
# Full Hash: fe20f47f85d97b9c617f5a5c004887b08d130c5f
# Author: Botond Ballo <botond@mozilla.com>
# Date: 2022-07-19 09:34:24
# Regressor Bug: 1779144
# File Overlap Count: 1
# Description:
#   Bug 1779144 - Handle touchpad double-taps correctly in APZCTreeManager::ConvertToGecko(). r=dlrobertson
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D151939
# ==============================================================================

diff -r df2118afa498 -r fe20f47f85d9 gfx/layers/apz/src/APZCTreeManager.h
--- a/gfx/layers/apz/src/APZCTreeManager.h	Tue Jul 19 00:56:45 2022 +0300
+++ b/gfx/layers/apz/src/APZCTreeManager.h	Mon Jul 18 21:37:47 2022 +0000
@@ -987,6 +987,11 @@
   /* Tracks the number of touch points we are tracking that are currently on
    * the screen. */
   TouchCounter mTouchCounter;
+  /* If a tap gesture event sent directly by widget code (rather than gesture
+   * detected from touch events by APZ) is being processed, this stores the
+   * result of hit testing for that tap gesture event.
+   */
+  HitTestResult mTapGestureHitResult;
   /* Stores the current mouse position in screen coordinates.
    */
   mutable DataMutex<ScreenPoint> mCurrentMousePosition;
